astropy/time/formats.py

```diff
@@ -745,6 +746,52 @@ class TimeUnixTai(TimeUnix):
     epoch_scale = 'tai'
 
 
+class TimeUnixPTP(TimeUnix):
+    """
+    Counts the number of SI seconds elapsed since 1970-01-01 00:00:00 TAI.
+
+    This is basically the same as the ``unix_tai`` with an offset of 8 seconds
+    due to the different epoch (1970-01-01 00:00:00 UTC vs. TAI).
+
+    This will generally differ from Unix time by the cumulative integral number
+    of leap seconds since 1970-01-01 UTC plus the initial offset of 8 seconds.
+
+    This convention matches the definition of the Precision Time Protocol,
```

Comment:
Put this paragraph first in the docstring as the fundamental definition of the format, rather than starting with the format just being an 8-second derivative of `unix_tai`.

Reply:
done